workflow_tests/lint.test.ts

```diff
@@ -1,12 +1,13 @@
-const path = require('path');
-const kieMockGithub = require('@kie/mock-github');
-const utils = require('./utils/utils');
-const assertions = require('./assertions/lintAssertions');
-const mocks = require('./mocks/lintMocks');
-const ExtendedAct = require('./utils/ExtendedAct').default;
+import {MockGithub} from '@kie/mock-github';
+import path from 'path';
+import assertions from './assertions/lintAssertions';
+import LINT__LINT__STEP_MOCKS from './mocks/lintMocks';
```

Comment:
```suggestion
import mocks from './mocks/lintMocks';
```

Merge with `main` first